### PR TITLE
Another style for captions

### PR DESCRIPTION
**Pull request by drmenzelit**
Pull Request for Issue #157  .

Alternative to PR #160 - testers report back which one you prefer.

### Summary of Changes
Small font, light gray and more space between image and intro text

### Testing Instructions
Run npm

### CODE REVIEW

File: templates/cassiopeia/scss/blocks/_global.scss [hunk 1]
```diff
@@ -64,6 +64,15 @@ th dd {
   font-weight: normal;
 }
 
+figure {
+  margin: 0 0 ($cassiopeia-grid-gutter*2);
```
Comment (hans2103) [inline templates/cassiopeia/scss/blocks/_global.scss:68]: using the wildcard selector (previously known as lobotomized owl selector) :-) this margin-bottom is not a good idea. Can we remove the margin-bottom on all and only use margin-top on succeeding elements?

**Comment by richard67** [comment]
To me this PR here looks better on sample data than the other PR #160 .

### This PR here:


### Other PR #161 :


But that's a matter of taste, I'm ok with both.

**Comment by richard67** [comment]
But the additional margin buttom is not really visible, so I think it might really be removed.

**Comment by chmst** [comment]
After testing with different views - we can take this one if the commen of @hans2103 is answered.

File: templates/cassiopeia/scss/blocks/_global.scss (hunk 1 above)
Comment (drmenzelit) [reply templates/cassiopeia/scss/blocks/_global.scss:68]: That would mean to add margin-top to the title of the article, right?

Comment (hans2103) [reply templates/cassiopeia/scss/blocks/_global.scss:68]: I mean : https://every-layout.dev/layouts/stack/

**Comment by hans2103** [comment]
what I mean is : https://every-layout.dev/layouts/stack/

I prefer this PR above #160
And according to our Glip discussion both @chmst and @richard67 do too.

**Review by hans2103 — APPROVED** [review]

**Pull request merged**